cli/command/image/load_test.go

```diff
@@ -113,6 +113,22 @@ func TestNewLoadCommandSuccess(t *testing.T) {
 				return image.LoadResponse{Body: io.NopCloser(strings.NewReader("Success"))}, nil
 			},
 		},
+		{
+			name: "with-comma-separated-platforms",
+			args: []string{"--platform", "linux/amd64,linux/arm64/v8,linux/riscv64"},
+			imageLoadFunc: func(input io.Reader, options ...client.ImageLoadOption) (image.LoadResponse, error) {
+				assert.Check(t, len(options) > 0) // can be 1 or two depending on whether a terminal is attached :/
```

Comment:
Hmm I'm not sure I understand this one. How does terminal affect parsing here?

Reply:
Ah right.. We should apply these opts to a temporary var and then check the resulting opts struct, but we can't do that because these types are unexported..

I guess the only way to test it in this situation is by checking HTTP request at the HTTP client, but that would need rewriting the test a bit.

Although, I think the best solution would be to have a debug helper on the client side that would allow introspecting for testing purposes...